pvacseq/lib/pipeline.py

```diff
@@ -51,6 +57,18 @@ def convert_vcf(self):
             convert_params.extend(['-g', self.gene_expn_file])
         if self.transcript_expn_file is not None:
             convert_params.extend(['-i', self.transcript_expn_file])
+        if self.normal_snvs_coverage_file is not None:
+            convert_params.extend(['--normal-snvs-coverage-file', self.normal_snvs_coverage_file])
+        if self.normal_indels_coverage_file is not None:
+            convert_params.extend(['--normal-indels-coverage-file', self.normal_indels_coverage_file])
+        if self.tdna_snvs_coverage_file is not None:
+            convert_params.extend(['--tdna-snvs-coverage-file', self.tdna_snvs_coverage_file])
+        if self.tdna_indels_coverage_file is not None:
+            convert_params.extend(['--tdna-indels-coverage-file', self.tdna_indels_coverage_file])
+        if self.trna_snvs_coverage_file is not None:
+            convert_params.extend(['--trna-snvs-coverage-file', self.trna_snvs_coverage_file])
+        if self.trna_indels_coverage_file is not None:
+            convert_params.extend(['--trna-indels-coverage-file', self.trna_indels_coverage_file])
```

Comment:
Is there a good way to do these like:
```perl
for my $type (qw(tnra_indels_coverage_file ...)) {
  if (my $value = defined $self->$type) {
    my $param = '--' . $type;
    $param =~ tr/_/-/;
    push @covert_params, $param, $value;
  }
}
```
to avoid needing to repeat the if block for each file we might have?